jni/BUILD.adoc

```diff
@@ -102,6 +118,10 @@ The instructions below assume that you start each build in `/path/to/kotlin-satl
 
  make jcadical CADICAL_INSTALL_DIR=solvers/cadical-src/install
 
+* 👨🏻‍💻 On macOS (`libjcadical.dylib`)
+
+ make jcadical CADICAL_INSTALL_DIR=solvers/cadical-src/install LIB_EXT=dylib JAVA_INCLUDE_SUBDIR=darwin JAVA_HOME=$(/usr/libexec/java_home) JCADICAL_CXXFLAGS="-std=c++11"
```

Comment:
Please, fix the order of args: group all java-related stuff first, then group all the cadical-related stuff, etc.